tests/test_timerange.py

```diff
@@ -1,11 +1,16 @@
 import datetime
+import os
 import unittest
+from unittest.mock import patch
 
 from datagrepper.util import assemble_timerange
 
 
 utc = datetime.timezone.utc
 
+with patch.dict(os.environ, DATAGREPPER_CONFIG="/dev/null"):
+    import datagrepper.app
```

Comment:
This seems a bit wrong, loading the module should not fail if the environment variable is not defined in the first place. I think it would be better to protect the `config.from_envvar()` call with a `if VAR in os.environ:` in `app.py` instead.

Reply:
sorry -- will fix this -- i was just doing what was already being done in the other tests